### PR TITLE
Getting Started Python client

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Hey @chaudum as you are the Python product owner, can you let me know if you are happy with this very basic overview of how to install driver and connect to a cluster with Python?

It's just for the Getting Started workflow / overview.

### CODE REVIEW

**Comment by ChrisChinchilla** [comment]
@chaudum Did you send me the link to this gist - https://gist.github.com/chaudum/7821fae242012fc42054

For this page? Or just for interest :)

File: pages/docs/getting-started/clients/python.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+```python
+from crate import client
+```
+## Connect to Crate
+
+Use the `client` module to create a connection:
+
+```python
+connection = client.connect(SERVER_IP)
```
Comment (chaudum) [inline pages/docs/getting-started/clients/python.html]:
```python
# obtain a cursor
cursor = connection.cursor()
# execute a sql statement
cursor.execute('SELECT name FROM sys.cluster')
# fetch results
result = cursor.fetchone()
```

Comment (chaudum) [inline pages/docs/getting-started/clients/python.html]: also, you could add the the python client is an implementation of [PEP0249](https://www.python.org/dev/peps/pep-0249/)

Comment (ChrisChinchilla) [inline pages/docs/getting-started/clients/python.html]: Yes, @chaudum ChrisC didn't want any queries in these opening docs, but I think I might add the cursor line at the very least as that is kind of still 'making a connection'.

**Comment by ChrisChinchilla** [comment]
Merging as part of one large PR.

**Pull request closed without merging**